src/components/questionary/questionaryComponents/Interval/QuestionNumberInputForm.tsx

```diff
@@ -0,0 +1,117 @@
+import makeStyles from '@material-ui/core/styles/makeStyles';
```

Comment:
Not sure why this `QuestionNumberInputForm` is inside `Interval` folder. I see the same file name inside `NumberInput` form. Or maybe it is not the same but correct me if I am wrong.

Reply:
removed unused file